### PR TITLE
first implementation

### PR DESCRIPTION
**Pull request by michaelwittig**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+  }
+
+  if (del === true) {
+    await deleteAMIs(ec2, amis);
+  } else {
+    return;
+  }
+}
+
+function readArgs() {
```
Comment (andreaswittig) [inline index.js:97]: How to configure the region?

**Review by andreaswittig — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (michaelwittig) [reply index.js:97]: The only wait is to set the `AWS_REGION` environment variable at the moment. Should we add and arg?

Comment (andreaswittig) [reply index.js:97]: I'd say it would be worth the effort.

Comment (michaelwittig) [reply index.js:97]: @andreaswittig done

**Review by andreaswittig — APPROVED** [review]

**Pull request merged**